core/commands/partials/vagrant_configurator.rb

```diff
@@ -238,9 +238,15 @@ def up
     nodes = @config.node_names
     run_in_directory(@config.path) do
       store_network_config
-      up_results = Workers.map(nodes) { |node| up_node(node) }
-      up_results.each { |up_result| up_result[1].print_to_stdout } if @env.threads_count > 1
-      return ERROR_RESULT unless up_results.detect { |up_result| !up_result[0] }.nil?
+      if nodes.size != 1
```

Comment:
The logic for such check is currently in method `retrieve_logger_for_node` and on line 242.

You must calculate the need for modified behavor beforehand and use the logical variable in both places.